Python/gc.c

```diff
@@ -1709,20 +1709,25 @@ _PyGC_GetObjects(PyInterpreterState *interp, int generation)
     GCState *gcstate = &interp->gc;
 
     PyObject *result = PyList_New(0);
-    if (result == NULL) {
-        return NULL;
+    /* Generation:
+     * -1: Return all objects
+     * 0: All young objects
+     * 1: No objects
+     * 2: All old objects
+     */
+    if (result == NULL || generation == 1) {
```

Comment:
As a procedural question, do we need to seek release manager permission for this on 3.13 following the release candidate?

Reply:
No harm in letting @Yhg1s know now. He'll definitely need to know for the backport.